src/components/hv-text-field/index.js

```diff
@@ -34,6 +34,8 @@ const HvTextField = (props: HvComponentProps) => {
 
   // Extract known attributes into their own variables
   const autoFocus = props.element.getAttribute('auto-focus') === 'true';
+  const debounceTime =
```

Comment:
Maybe call this `debounceMs` to clarify the units?

Reply:
done in [74ea644](https://github.com/Instawork/hyperview/pull/664/commits/74ea644f1fc3846d198ce23419971f3e6ceec60e)